components/tests/python/library/__init__.py

```diff
@@ -116,6 +116,18 @@ def teardown_class(cls):
         cls.root = None
         cls.__clients.__del__()
 
+    def keepRootAlive(self):
+        """
+        Keeps root connection alive.
+        """
+        try:
+            if self.root.sf is None:
+                assert self.root.connect()
```

Comment:
Yes, sorry, this is bad pasting from me and because nothing failed not looking further.